CorsixTH/Lua/languages/english.lua

```diff
@@ -260,6 +260,8 @@ adviser = {
     th_cheat = "Congratulations, you have unlocked cheats!",
     roujin_on_cheat = "Roujin's challenge activated! Good luck in the coming months...",
     roujin_off_cheat = "Roujin's challenge deactivated. Everything will be back to normal soon.",
+    norest_on_cheat = "It appears you staff have overdosed on caffeine and no longer feel a need to rest.",
```

Comment:
```suggestion
    norest_on_cheat = "It appears your staff have overdosed on caffeine and no longer feel a need to rest.",
```

Reply:
I'd also like to open the floor to opinions on using `overdosed` -- the term I feel shouldn't be used with jest personally but had nothing better to fit at the time. Maybe I'm thinking too hard on it :)

Reply:
```suggestion
    norest_on_cheat = "Oh no! It appears your staff have consumed too much caffeine and no longer feel a need to rest.",
```